core/templates/components/lockup/lockup.twig

```diff
@@ -0,0 +1,67 @@
+{#
+/**
+ * @file
+ * Lockup Component.
+ *
+ * A lockup with the Stanford wordmark logo and department/site name. It links to the homepage of the department/unit.
+ *
+ * Available variants for the lockup:
+ * - .su-lockup - Lockup with one line to the right of the wordmark.
+ * - .su-lockup--two-lines - Lockup with two lines to the right of the wordmark.
+ *
+  * Available variants for the individual site name lines:
+ * - .su-lockup__sitename-line1 - Regular font style for line 1
+ * - .su-lockup__sitename-line1-small - Small font size for line 1
+ * - .su-lockup__sitename-line1-supersmall - Supersmall font size for line 1
+ * - .su-lockup__sitename-line1-uppercase - Uppercase and bold font for line 1
+ * - .su-lockup__sitename-line2 - Regular font style for line 2
+ * - .su-lockup__sitename-line5 - Regular font style for line 5
+ * - .su-lockup__sitename-line5-uppercase - Uppercase, bold and smaller font for line 5
+ *
```

Comment:
I see what you mean, @JBCSU . My proposed tweak would be to list the "defaults" in their own section, and then the variants. @josephgknox and @sherakama , LMK what you think.

Reply:
I redid the comments in lockup.twig. LMK what you all think.

Reply:
Thanks for bringing this up @JBCSU. I completely agree with you here. @yvonnetangsu thanks for addressing this already. My one (very small) comment would be to move the **Default classes for the individual site name lines** comment above the **Available variant for the lockup** comment so that class comments and variant comments are next to one another.